README.md

```diff
@@ -333,8 +339,8 @@ In this format, "settings.kubernetes.cluster-name" refers to the same key as in
 
 The following settings must be specified in order to join a Kubernetes cluster.
 You should [specify them in user data](#using-user-data).
-See the [setup guide](QUICKSTART.md) for *much* more detail on setting up Bottlerocket and Kubernetes.
-* `settings.kubernetes.cluster-name`: The cluster name you chose during setup; the [setup guide](QUICKSTART.md) uses "bottlerocket".
+See the [setup guide](QUICKSTART-EKS.md) for *much* more detail on setting up Bottlerocket and Kubernetes.
```

Comment:
nit: It won't let me leave a suggestion for whatever reason, but upon reading this and the ECS version in context, I think the "see the guide" note belongs above the "the following settings..." note; the note applies to the whole "Kubernetes/ECS settings" section, not just the first group of settings.  (And if we're changing it, I'd get rid of the `*`...)

Reply:
Done